esp/src/src-react/components/forms/landing-zone/BlobImportForm.tsx

```diff
@@ -144,7 +144,7 @@ export const BlobImportForm: React.FunctionComponent<BlobImportFormProps> = ({
                 newValues.selectedFiles[idx] = {
                     TargetName: "",
                     SourceFile: file["fullPath"],
-                    SourcePlane: file?.DropZone?.Name ?? "",
+                    SourcePlane: file["SourcePlane"] ?? "",
```

Comment:
Is this consistent between bare metal and cloud?

Reply:
It was not. Corrected the definition of file["SourcePlane"] in LandingZone to ensure it accounts for isContainer.